apps/internal/base/base.go

```diff
@@ -458,6 +481,15 @@ func (b Client) AuthResultFromToken(ctx context.Context, authParams authority.Au
 	return ar, err
 }
 
+// This function wraps time.Now() and is used for refreshing the application
+// was created to test the function against refreshin
+var GetCurrentTime = time.Now
```

Comment:
1. Making sure this isn't part of the public API?
2. Does it make sense to keep it higher up?

Reply:
There is no other way to do that, I will consult with Charles on this again.

Reply:
While this is exported i.e. public and therefore accessible from other packages, it's in an internal package. Application code can't reference any internal package